internal/completions/httpapi/handler.go

```diff
@@ -73,12 +77,29 @@ func newCompletionsHandler(
 			Build()
 		defer done()
 
+		// Use the user's access token for Cody Gateway on dotcom if PLG is enabled
```

Comment:
Super-ulta pedantic nitpick. Please let me know if you never want to see PR suggestions like this in the future...

From Google's C++ style guide:

> Pay attention to punctuation, spelling, and grammar; it is easier to read well-written comments than badly written ones.
>
> Comments should be as readable as narrative text, with proper capitalization and punctuation. In many cases, complete sentences are more readable than sentence fragments. Shorter comments, such as comments at the end of a line of code, can sometimes be less formal, but you should be consistent with your style.
>
> Although it can be frustrating to have a code reviewer point out that you are using a comma when you should be using a semicolon, it is very important that source code maintain a high level of clarity and readability. Proper punctuation, spelling, and grammar help with that goal.
> https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar

```suggestion
		// Use the user's access token for Cody Gateway on dotcom if PLG is enabled.
```

Reply:
I agree with the sentiment, but we don't have this in our coding style guide (which we also don't quite have, btw. We have [this](https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/style_and_mechanics/) and Prettier + ES Lint rules, but no real coding style guide), and as a result, we have a bunch of comments which are sentence fragments. I prefer complete sentences that have a beginning and end, proper capitalization, and punctuation, but I also want to be idiomatic with the rest of the codebase. This often makes me wonder about stuff like periods at the end of comments.

Reply:
In short, hopefully it isn't too much trouble to just click "Add suggestion to batch" and not worry about it :)

But as far as this not being a _requirement_ at Sourcegraph, thanks for calling that out. I'll try to be less vocal about this particular class of code review feedback. It's something I feel stronger than I ought to about, but at the same time am 100% clear in saying it generates "light but no heat". But in the spirit of making code as clear and high-quality as possible, IMHO, it's worth calling out. (But if you disagree or finding this type of PR feedback annoying, I can certainly not mention it, since I know it can be irksome.)